packages/n4s/src/enforce/runner/index.js

```diff
@@ -11,8 +11,17 @@ function runner(rule, value, ...args) {
     return;
   }
 
-  if (rule(value, ...args) !== true) {
-    throw new Error(`[Enforce]: invalid ${typeof value} value`);
+  const ruleResult = rule(value, ...args);
+  // Handles boolean rules
+  if (typeof ruleResult === 'boolean' && ruleResult !== true) {
```

Comment:
I believe this condition covers the same cases.
```suggestion
  if (ruleResult === false) {
```